test/WebJobs.Script.Tests/ScriptHostTests.cs

```diff
@@ -1056,16 +995,88 @@ public void ValidateFunction_ValidatesHttpRoutes()
             });
             Assert.Equal("The specified route conflicts with one or more built in routes.", ex.Message);
 
-            // verify that empty route is defaulted to function name
```

Comment:
These existing tests were commented out unecessarily - I added a new test case for "runtime". We already had a case for "admin"